File: cpp_solutions/1550.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int cnt = 0;
        for (auto& num : arr) {
            if (num & 1) {
                cnt += 1;
            }
            else cnt = 0;
            if (cnt == 3) return true;
        }
        return false;
    }
};

File: rust_solutions/1550.rs
impl Solution {
    pub fn three_consecutive_odds(arr: Vec<i32>) -> bool {
        let mut cnt = 0;
        for num in arr {
            if (num & 1) == 1 {
                cnt += 1;
            }
            else {
                cnt = 0;
            }
            if cnt == 3 {
                return true;
            }
        }
        return false;
    }
}
